include/circt/Dialect/RTL/Structure.td

```diff
@@ -61,6 +58,57 @@ def RTLModuleOp : RTLOp<"module",
   let parser = [{ return ::parse$cppClass(parser, result); }];
 }
 
+def RTLExtModuleOp : RTLOp<"extmodule",
+      [IsolatedFromAbove, FunctionLike, Symbol]> {
+  let summary = "RTL extModule";
+  let description = [{
+    The "rtl.extmodule" operation represents an external reference to a Verilog
+    module, including a given name and a list of ports.
+  }];
+  let arguments = (ins OptionalAttr<StrAttr>:$defname,
+                       OptionalAttr<DictionaryAttr>:$parameters);
+  let results = (outs);
```

Comment:
Should we start here with the 'outputs as results' changes or keep this consistent with the current RTL module? I can see arguments either way.